chaoswm/actions.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+from typing import Any, List, Mapping
+
+from chaoslib.types import Configuration
+from logzero import logger
+
+from .driver import Wiremock
+from .utils import check_configuration, get_wm_params
+
+__all__ = [
+    "add_mappings",
+    "populate_from_dir",
+    "delete_mappings",
+    "global_fixed_delay",
+    "global_random_delay",
+    "fixed_delay",
+    "random_delay",
+    "chunked_dribble_delay",
+    "down",
+    "up",
+    "reset"
+]
+
+
+def add_mappings(
+        mappings: List[Any],
+        configuration: Configuration = None) -> List[Any]:
+    """ adds more mappings to wiremock
+    returns the list of ids of the mappings added
+    """
+    if not check_configuration(configuration):
+        return []
+
+    params = get_wm_params(configuration)
+    w = Wiremock(url=params['url'], timeout=params['timeout'])
+    return w.populate(mappings)
+
+
+def populate_from_dir(
+        dir: str = ".",
+        configuration: Configuration = {}) -> List[Any]:
```

Comment:
Missed this one:

`configuration: Configuration = {} `

I'm happy to fix after the merge.